src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets

```diff
@@ -0,0 +1,231 @@
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+  <!--
+    ============================================================
+                                        _RunOptimizer
+
+    Start the optimization phase
+    ============================================================
+    -->
+  <Target Name="_RunOptimizer"
+          DependsOnTargets="_ComputeResolvedFilesToCacheTypes;
+                            _GenerateCrossgenProj;
+                            _SetupCrossgenStage;
+                            _RestoreCrossgen"
+          Condition="$(SkipOptimization) != 'true' ">
+    <!-- Get the coreclr path -->
+    <ItemGroup>
+      <_CoreclrResolvedPath Include="@(CrossgenResolvedAssembliesToPublish)"
+                                             Condition="'%(CrossgenResolvedAssembliesToPublish.Filename)'=='coreclr'" />
+      <_CoreclrResolvedPath Include="@(CrossgenResolvedAssembliesToPublish)"
+                                             Condition="'%(CrossgenResolvedAssembliesToPublish.Filename)'=='libcoreclr'" />
+      <_JitResolvedPath Include="@(CrossgenResolvedAssembliesToPublish)"
+                                             Condition="'%(CrossgenResolvedAssembliesToPublish.Filename)'=='clrjit'" />
+      <_JitResolvedPath Include="@(CrossgenResolvedAssembliesToPublish)"
+                                             Condition="'%(CrossgenResolvedAssembliesToPublish.Filename)'=='libclrjit'" />
+      <_AssembliestoCrossGen Include="$(_RuntimeDir)\**\*.*" />
+    </ItemGroup>
+
+    <NETSdkError Condition="'@(_CoreclrResolvedPath->Count())' &gt; 1"
+    ResourceName="MultipleFilesResolved"
+    FormatArguments="coreclr" />
+
+    <NETSdkError Condition="'@(_CoreclrResolvedPath)'== ''"
+    ResourceName="UnableToFindResolvedPath"
+    FormatArguments="coreclr" />
+
+    <NETSdkError Condition="'@(_JitResolvedPath->Count())' &gt; 1"
+    ResourceName="MultipleFilesResolved"
+    FormatArguments="jit" />
+
+    <NETSdkError Condition="'@(_JitResolvedPath)'== ''"
+    ResourceName="UnableToFindResolvedPath"
+    FormatArguments="jit" />
+    
+    
+    
+    <!-- Get the crossgen and jit path-->
+    <PropertyGroup>
+      <_CoreclrPath>@(_CoreclrResolvedPath)</_CoreclrPath>
+      <_JitPath>@(_JitResolvedPath)</_JitPath>
+      <_CoreclrDir>$([System.IO.Path]::GetDirectoryName($(_CoreclrPath)))</_CoreclrDir>
+      <_CoreclrPkgDir>$([System.IO.Path]::Combine($(_CoreclrDir),"..\..\..\"))</_CoreclrPkgDir>
```

Comment:
Because in "2.0", Microsoft.NETCore.App no longer depends on the CoreCLR package.  Instead, it takes all the assets out of the CoreCLR package (and all the CoreFX packages) and puts them directly in the Microsoft.NETCore.App package.